envelope.go

```diff
@@ -34,6 +35,59 @@ func (e *Envelope) GetHeader(name string) string {
 	return decodeHeader(e.header.Get(name))
 }
 
+// GetHeaderValues processes the specified header for RFC 2047 encoded words and returns all existing
+// values as a list of UTF-8 strings
+func (e *Envelope) GetHeaderValues(name string) []string {
+	if e.header == nil {
+		return []string{}
+	}
+
+	h := *e.header
```

Comment:
Cannot get rid of the `h := *e.header` line, because you cannot index pointers. Otherwise I implemented your suggestion.

Reply:
I think you can do it with parenthesis, at least this works in the go playground: https://play.golang.org/p/dBLMFYZYPrn

and my comments on the tests (using Got:) wasn't for style reasons, it was because it says `To` but those lines were testing headers other than `To`

Reply:
👍